src/main/java/codesquad/springcafe/article/ArticleController.java

```diff
@@ -0,0 +1,55 @@
+package codesquad.springcafe.article;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.Optional;
+
+@Controller
+public class ArticleController {
+    private final Logger log = LoggerFactory.getLogger(getClass());
+    private final ArticleRepository articleRepository;
+
+    public ArticleController(ArticleRepository repository) {
+        articleRepository = repository;
+    }
+
+    @GetMapping("/article/{id}")
+    public String showArticle(@PathVariable int id, Model model) {
+        Optional<Article> optArticle = articleRepository.findBy(id);
+        return optArticle.map(article -> {
+            model.addAttribute("article", article);
+            return "qna/show";
+        }).orElse("index");
+    }
+
+
+    @PostMapping("/article")
+    public String storeArticle(@ModelAttribute ArticleDto articleDto) {
+        final String writer = articleDto.getWriter();
+        final String title = articleDto.getTitle();
+        final String contents = articleDto.getContents();
+        final LocalDateTime createAt = LocalDateTime.now();
+        final int id = articleRepository.size() + 1;
+        Article article = new Article(id, writer, title, contents, createAt);
```

Comment:
```suggestion
        Article article = Article.from(articleDto);
```

요렇게 한 번에 만들면 어떨까 싶기도 하네요.